python/torch_mlir_e2e_test/test_suite/reduction.py

```diff
@@ -315,6 +315,44 @@ def ReduceMaxAlongDim_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ReduceMaxAlongDimSignedInt(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.int64, True),
+    ])
+    def forward(self, a):
+        return torch.ops.aten.max(a, 1)[0]
```

Comment:
Why only return the first element of the result?

Reply:
Fixed